libmachine/drivers/plugin/plugin.go

```diff
@@ -84,7 +84,7 @@ func NewLocalBinaryPlugin(driverName string) *LocalBinaryPlugin {
 func (lbe *LocalBinaryExecutor) Start() (*bufio.Scanner, *bufio.Scanner, error) {
 	log.Debugf("Launching plugin server for driver %s", lbe.DriverName)
 
-	binaryPath, err := exec.LookPath(fmt.Sprintf("docker-machine-%s", lbe.DriverName))
+	binaryPath, err := exec.LookPath(fmt.Sprintf("docker-machine-driver-%s", lbe.DriverName))
```

Comment:
I'm leaning pretty heavily towards sticking with `docker-machine-foo` for simplicity and brevity's sake.